plugin/trace/setup_test.go

```diff
@@ -8,31 +8,34 @@ import (
 
 func TestTraceParse(t *testing.T) {
 	tests := []struct {
-		input        string
-		shouldErr    bool
-		endpoint     string
-		every        uint64
-		serviceName  string
-		clientServer bool
+		input           string
+		shouldErr       bool
+		endpoint        string
+		every           uint64
+		serviceName     string
+		clientServer    bool
+		serviceEndpoint string
```

Comment:
serviceEndpoint is empty in all tests? In general if you want to test a little new thing, just add a small new test.

Reply:
Fair.

Reply:
Removed it.